Dockerfile

```
@@ -47,4 +47,4 @@ USER 65534:65534
 ENV GRPC_GO_LOG_SEVERITY_LEVEL="INFO"
 
 COPY --from=builder /go/node-feature-discovery/deployment/components/worker-config/nfd-worker.conf.example /etc/kubernetes/node-feature-discovery/nfd-worker.conf
-COPY --from=builder /go/bin/* /usr/bin/
+COPY --from=builder /go/bin/grpc-health-probe /usr/bin/
```

Hmm, we should copy the nfd binaries too, right?

```suggestion
COPY --from=builder /go/bin/* /usr/bin/
```

---

@marquiz thank you for the review. That's interesting as I've tested this change before submitting PR. /usr/bin/nfd-master and /usr/bin/nfd-worker binaries present in the image. Do they come there from the base image?

---

Here is what I can see in the running nfd-master container:
```
nobody@nfd-master-79bf6776fd-cn586:/$ ls -la /usr/bin/grpc*
-rwxr-xr-x 1 root root 13659507 Dec 22 14:33 /usr/bin/grpc-health-probe

nobody@nfd-master-79bf6776fd-cn586:/$ ls -la /usr/bin/nfd-*
-rwxr-xr-x 1 root root 34410496 Dec 22 14:37 /usr/bin/nfd-master
-rwxr-xr-x 1 root root 35565568 Dec 22 14:37 /usr/bin/nfd-topology-updater
-rwxr-xr-x 1 root root 18563072 Dec 22 14:37 /usr/bin/nfd-worker
```

---

> @marquiz thank you for the review. That's interesting as I've tested this change before submitting PR. /usr/bin/nfd-master and /usr/bin/nfd-worker binaries present in the image. Do they come there from the base image?

We have "normal" and "minimal" image variants. You changed the minimal variant so the normal image worked fine